pull_request_template.md

```diff
@@ -9,45 +9,62 @@ Closes: #XXX
 
 ## What is the purpose of the change
 
-> Add a description of the overall background and high level changes that this PR introduces
+<!-- Add a description of the overall background and high level changes that this PR introduces
 
-*(E.g.: This pull request improves documation of area A by adding ....*
+*(E.g.: This pull request improves documation of area A by adding ....* -->
 
 
 ## Brief Changelog
 
-*(for example:)*
+<!-- *(for example:)*
  
   - *The metadata is stored in the blob store on job creation time as a persistent artifact*
   - *Deployments RPC transmits only the blob storage reference*
-  - *Daemons retrieve the RPC data from the blob cache*
+  - *Daemons retrieve the RPC data from the blob cache* -->
 
 
-## Testing and Verifying
+## Author's Checklist
 
-*(Please pick one of the following options)*
+I have...
 
-This change is a trivial rework / code cleanup without any test coverage.
```

Comment:
Given these take ~30 min right now, it might be a bit unrealistic to run these for every PR

There are also trivial PRs where we wouldn't have to run tests (e.g. changes that don't change core chain logic), so we might want to keep this line around

Reply:
good call, done

Reply:
As annoying as it is, I'd advocate for running the integration tests before each merge. I wasted half the day yesterday fixing things that broke across multiple PRs. It would have been much easier to fix each at the time of the change. And most of them wound up being changes that you would not have expected to break the tests (e.g. removing a log).

I think we should up the priority of getting these running in GH actions so we can offload the burden and context switch to a different ticket. I'm happy to tackle this next week

Reply:
Agree re pushing these into GH actions!

Reply:
@asalzmann what's your take on the priority level of getting those into GH actions?

Reply:
If we run these after every PR, I think we're going to slow down our dev cycle pretty significantly, although I hear you on having to go back and fix the tests, and that being really annoying (also shouldn't fall on you - if integration tests are broken, we should put it in #development and figure out why together).

What if we compromised and ran the gaia tests after every PR, until we can get these running in actions? That feels like the 80/20